serrano/resources/field/base.py

```diff
@@ -82,7 +85,14 @@ def get_link_templates(self, request):
         }
 
     def get_queryset(self, request, params):
-        queryset = self.model.objects.all()
+        tree = trees[params['tree']]
+        q = Q()
+
+        # No public method for accessing the local models on the tree
+        for app_name, model_name in tree._models:
+            q |= Q(app_name=app_name, model_name=model_name)
+
+        queryset = self.model.objects.filter(q)
```

Comment:
Any reason you create the `Q` conditions by ORing and filtering here but ANDing and excluding in the Concept `get_queryset` method above? If there is no reason for the difference then it might be good to construct and use the `Q` conditions the same way in both Field and Concept cases(nit). If there is a reason for the difference, might be worth documenting somewhere.